File: server/go/core/test/grpc/sigma/Tower.ts
// Original file: packets.proto

import type { Long } from '@grpc/proto-loader';

export interface Tower {
  'id'?: (number | string | Long);
  'name'?: (string);
  'avatarId'?: (number | string | Long);
  'isPublic'?: (boolean);
  'creatorId'?: (number | string | Long);
}

export interface Tower__Output {
  'id': (string);
  'name': (string);
  'avatarId': (string);
  'isPublic': (boolean);
  'creatorId': (string);
}

File: servers/main/core/models/tower.go
package models

type Tower struct {
	Id        int64  `json:"id"`
	Name      string `json:"name"`
	AvatarId  int64  `json:"avatarId"`
	IsPublic  bool   `json:"isPublic"`
	CreatorId int64  `json:"creatorId"`
}
